frontend/views/containers/proposals/Mincome.vue

```diff
@@ -1,61 +1,35 @@
 <template lang='pug'>
   proposal-template(
     :title='L("Change minimum income")'
-    :footer='L("According to your voting rules, 8 out of 10 members will have to agree with this.")'
+    :rule='{ value: 8, total: 10 }'
     :disabled='$v.form.$invalid || ($v.steps[config.steps[currentStep]] && $v.steps[config.steps[currentStep]].$invalid)'
     :maxSteps='config.steps.length'
     :currentStep.sync='currentStep'
-    @submit='L("submit")'
+    @submit='submit'
   )
 
-    .field(v-if='currentStep === 0' key='0')
-      i18n.label(tag='label') New minimum income
-      .select-wrapper
+    label.field(v-if='currentStep === 0' key='0')
+      i18n.label New minimum income
+      .input-combo
```

Comment:
Down below, boy scoutt:

```js
:placeholder='L("Amount")'
```

Reply:
Is this a value for the placeholder? If it is, I would rather it stay empty ass specified in the designs.

Reply:
If we use a placeholder, at least have a value of the same type as the final value, for example, "500" in this case. But in this case in specific, I would prefer to leave it empty too.

Reply:
That might even be more dangerous @sandrina-p - the user might think that the field is already filled!

Reply:
Removed all placeholders